### DISCUSSION TITLE
Tag Support

### DISCUSSION THREAD

**Discussion by bionemesis** [dec0dOS/zero-ui#19] [General]
Does zero-ui support tags? We have a flow rule we use in ZeroTier Central that restricts all communication between devices unless originating from specific devices that are tagged as servers. We'd like to do the same in zero-ui. Thank you.

**Comment by dec0dOS** [comment]
Hello, @bionemesis!
Currently, ZeroUI tags UI is in progress, you could define the rules via rules editor, but not the tags. Tags could be set from API for now.

**Reply by bionemesis** [reply]
Any suggestions on how to set the tags via the API? Not sure if there is API documentation or not, sorry.

**Reply by dec0dOS** [reply]
Please check the `update network configuration` POST request.
https://docs.zerotier.com/central/v1/#operation/updateNetwork

**Reply by dec0dOS** [reply]
Hello, @bionemesis.
Tags support has been added in 1.1.0 (https://github.com/dec0dOS/zero-ui/blob/main/CHANGELOG.md)
